### PR TITLE
RUBY-1236 Parse localLogicalSessionTimeoutMinutes from isMaster response

### PR DESCRIPTION
**Pull request by estolfo**

### CODE REVIEW

**Comment by estolfo** [comment]
Hi @ajdavis 
Would you mind looking this over? Thank you

File: lib/mongo/cluster.rb [hunk 1]
```diff
@@ -442,6 +442,21 @@ def addresses
...
+    #
+    # @example Get the logical session timeout in minutes.
+    #   cluster.logical_session_timeout
+    #
+    # @return [ Integer, nil ] The logical session timeout.
+    #
+    # @since 2.5.0
+    def logical_session_timeout
+      servers.inject(nil) do |min, server|
+        break unless timeout = server.logical_session_timeout
```
Comment (ajdavis) [inline lib/mongo/cluster.rb]: This returns nil if any server's timeout is nil?

Comment (estolfo) [reply lib/mongo/cluster.rb]: Yes. The ``break`` ends the iteration returning nil from the block.

Comment (ajdavis) [reply lib/mongo/cluster.rb]: Great, looks good to me.

Comment (estolfo) [reply lib/mongo/cluster.rb]: thanks

**Pull request merged**